Scripts/Source/UD_CustomDevice_RenderScript.psc

```diff
@@ -2829,188 +3123,552 @@ EndFunction
 ;==============================================================================================
 ;==============================================================================================
 
-;/  Group: Minigame
+
+;/  Group: Device Menu
 ===========================================================================================
 ===========================================================================================
 ===========================================================================================
 /;
 
-;/  Function: getDurabilityDmgMod
-    This value might not be correct. It is only updated when device menu is opened and minigame starts
-
-    Returns:
-
-        Device durability dmg after after difficulty is applied
-/;
-float Function getDurabilityDmgMod()
-    return _durability_damage_mod
-EndFunction
-
-;/  Function: updateDifficulty
-    Update device difficulty. Updates return value of <getDurabilityDmgMod>. Validate jammed locks
+;/  About: Control Array
+    Control array is used for filtering menu options. The value is inverted. So if you want to disable one of the options, you have to change nth value to True
+    
+    Defaultly all array values are False, so if you create the array, you can only disable options that you don't want, while ignoring other options
```

Comment:
All array values set to False by default, so...

Reply:
Fixed